app/javascript/src/navigation/uninav.js

```diff
@@ -31,25 +31,27 @@ const uninav = function() {
   function toggleAriaExpandVal() {
     $("#uninav li").on("click keypress", function (e) {
       if (a11yClick(event) === true) {
-        var menuItem = $(e.currentTarget);
+        var menuItem = $(e.currentTarget).find('.uninav__dropdown--child');
+
         if (menuItem.attr("aria-expanded") === "true") {
-          $(this).attr("aria-expanded", "false");
+          $(this).find('.uninav__dropdown--child').attr("aria-expanded", "false");
         } else {
-          $(this).attr("aria-expanded", "true");
+          $(this).find('.uninav__dropdown--child').attr("aria-expanded", "true");
         }
       }
     });
   }
 
   function hideCurrentDropdownWhenLoseFocus() {
     $(".uninav__dropdown--child li:last-of-type").on("keydown blur", function (e) {
+      debugger
```

Comment:
@cbryant24 heads up

Reply:
@nicholasnadel should be removed, good catch